### DISCUSSION TITLE
Is it possible to enable auto-tiling?

### DISCUSSION THREAD

**Discussion by vincentcomfy** [prasanthrangan/hyprdots#1859] [Q&A] [answered]
As the title suggests, is it possible to enable auto tiling similar to how it's managed in Pop!_OS? I don't really like that everything is tiled vertically with no indicator by default unless I press super J.

**Comment by kRHYME7** [comment]
No. 

Try asking the Hyprland repo 😅

Also Hyprland is a Dynamic Tiling Manager.

You can try to check the wiki though.
I guess you can make some rules to a specific workspace to behave into something.

**Reply by kRHYME7** [reply]
Can you describe how you want it to tile? Maybe we can help you set up the window rules

**Reply by vincentcomfy** [reply]
Sorry I forgot I posted this question, basically the behavior I'm looking for is where when opening a new window/app, instead of doing a vertical split by default, each app/window opens as a smaller division in a fibonnaci ratio.

The best example I can think of is how Pop!_OS handles tiling by default when enabled.

**Reply by vincentcomfy** [reply]
This is what I'm looking for https://youtu.be/-fltwBKsMY0?si=NeijmJrXeTjeTJmQ

**Reply by kRHYME7** [reply]
Check Both 

https://wiki.hyprland.org/Configuring/Dwindle-Layout/

Master layout looks similar to the popos one
https://wiki.hyprland.org/Configuring/Master-Layout/

You can check other layouts through plugins here.
https://wiki.hyprland.org/Configuring/Master-Layout/

**Reply by vincentcomfy** [reply]
Man you're always so helpful, it's shocking that all of this is free. Thank you, as always.

I'll try and figure it out based on what you shared.

**Reply by vincentcomfy** [reply]
Ah, so you're right in that what I'm actually looking for is a tiling layout, specifically "master".

I looked at the documentation under hyprland.conf and it looks like we're using dwindle by default.

I tried adding `workspace 1 rule = layout:master` in windowrules.conf, and `workspace_rule = workspace, layout:master` in hyprland.conf which _works_ but it leaves error messages saying that the config option doesn't exist.

How can I modify the tiling layout to just be master everywhere without the error messages?

**Reply by kRHYME7** [reply, marked as answer]
```

general:layout = master

```

Note that some properties,binds, etc are layout specific so you need to change yours that suits for master layout

**Reply by vincentcomfy** [reply]
> ```
> 
> general:layout = master
> ```
> 
> Note that some properties,binds, etc are layout specific so you need to change yours that suits for master layout

Lovely, this did the trick! Thank you so much, as always.
